lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js

```diff
@@ -27,18 +27,24 @@ function KeyboardShortcutsModal(props) {
   const shortcutsData = [];
```

Comment:
This piece of code from L27-L49 could probably made a bit more concise.

```suggestion
 const shortcutsData = allCommands.map(({ label, shortcut }) => {
      const shortcutCommand = shortcut.replace(/ /g, '').split('+');
      const commandArray = shortcutCommand.map(cmd => upperFirst(cmd.toLowerCase()));
      let camelCasedShortcut;

      if (platform.includes('Mac')) {
        camelCasedShortcut = commandArray.join(' + ').replace('Mod', 'Cmd').replace('Alt', 'Option');
      } else {
        camelCasedShortcut = commandArray.join(' + ').replace('Mod', 'Ctrl');
      }

      return {
        'action': label,
        'shortcut': camelCasedShortcut,
      };
    });
```